src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -89,67 +90,35 @@ public function getFilters()
     public function getTests()
     {
         return array(
-            new \Twig_SimpleTest('selectedchoice', array($this, 'isSelectedChoice')),
+            new \Twig_SimpleTest('selectedchoice', 'Symfony\Bridge\Twig\Extension\twig_is_selected_choice'),
         );
     }
 
     /**
      * {@inheritdoc}
      */
-    public function renderCsrfToken($tokenId)
-    {
-        return $this->renderer->renderCsrfToken($tokenId);
-    }
-
-    /**
-     * Makes a technical name human readable.
-     *
-     * @param string $text The text to humanize
-     *
-     * @return string The humanized text
-     */
-    public function humanize($text)
+    public function getName()
     {
-        return $this->renderer->humanize($text);
+        return 'form';
     }
+}
 
-    /**
-     * Returns whether a choice is selected for a given form value.
-     *
-     * Unfortunately Twig does not support an efficient way to execute the
-     * "is_selected" closure passed to the template by ChoiceType. It is faster
-     * to implement the logic here (around 65ms for a specific form).
-     *
-     * Directly implementing the logic here is also faster than doing so in
-     * ChoiceView (around 30ms).
-     *
-     * The worst option tested so far is to implement the logic in ChoiceView
-     * and access the ChoiceView method directly in the template. Doing so is
-     * around 220ms slower than doing the method call here in the filter. Twig
-     * seems to be much more efficient at executing filters than at executing
-     * methods of an object.
-     *
-     * @param ChoiceView   $choice        The choice to check
-     * @param string|array $selectedValue The selected value to compare
-     *
-     * @return bool Whether the choice is selected
-     *
-     * @see ChoiceView::isSelected()
-     */
-    public function isSelectedChoice(ChoiceView $choice, $selectedValue)
```

Comment:
This is clearly not an extension point, and as performance for this one is critical (the reason why we have this in the first place), I prefer to keep the function.